dv/uvm/icache/doc/ibex_icache_dv_plan.md

```diff
@@ -1,99 +1,142 @@
 ---
-title: "IBEX_ICACHE DV Plan"
+title: "Ibex ICache DV Plan"
 ---
 
 ## Goals
 * **DV**
-  * Verify all IBEX_ICACHE IP features by running dynamic simulations with a SV/UVM based testbench
-  * Develop and run all tests based on the [testplan](#testplan) below towards closing code and functional coverage on the IP and all of its sub-modules
+  * Verify all Ibex ICache IP features by running dynamic simulations with a SV/UVM based testbench
+  * Develop and run all tests based on the [testplan](#testplan) below towards closing code and functional coverage.
 * **FPV**
-
-## Current status
-<!-- TODO for now -->
-* [Design & verification stage]({{< relref "doc/project/hw_dashboard" >}})
-  * [HW development stages]({{< relref "doc/project/hw_stages" >}})
-* [Simulation results](https://reports.opentitan.org/hw/ip/ibex_icache/dv/latest/results.html)
+  * [TBD]
 
 ## Design features
-For detailed information on IBEX_ICACHE design features, please see the [IBEX_ICACHE technical specification]({{< relref "doc/icache.rst" >}}).
+
+The ICache design is documented in the [Instruction Cache](https://ibex-core.readthedocs.io/en/latest/icache.html) section of the Ibex User Manual.
 
 ## Testbench architecture
-IBEX_ICACHE testbench has been constructed based on the [DV_LIB testbench architecture]({{< relref "vendor/lowrisc_ip/dv_lib/" >}}).
 
-### Block diagram
-![Block diagram](tb.svg)
+The testbench intentionally avoids knowing detailed information about the cache's performance characteristics (for example, cache size, line size or number of ways).
+This means that the testbench cannot compare the DUT with a reference model, nor can it model the exact requests that the DUT will make of instruction memory: the whole point of a cache is that it might avoid an instruction fetch.
+
+The cache has two main interfaces, which face the core and instruction bus respectively.
+We model this with two agents: the *core agent* and the *memory agent*.
+The core agent will emulate a core making instruction fetch requests from the cache and the memory agent will emulate the instruction bus.
+
+In fact, there's one more logical interface: the `busy_o` signal from the cache.
+This signal is passed up to top-level in the design and warns the chip not to clock-gate or reset the core (because there are bus transactions in flight, or a memory invalidation in progress).
+Since this acts as a passive slave (rather than the core agent, which is an active master), we split this out into a very simple separate agent as well.
```

Comment:
If I'm understanding this, there will be a simple agent for just this one signal?
Would another option be to either include the `busy_o` signal in the iCache core interface or pass a generic interface for this signal into the icache core agent, since this signal is simply used to signal in-flight transactions or invalidations?

Reply:
That's a very good point. I was thinking that, since the `busy_o` signal is kind of a different interface from the rest of the cache <-> core signals, it should have a different agent, but I agree that it's slightly overkill. I'll push up a version of the document that gets rid of it.

Reply:
SGTM